Gemfile

```diff
@@ -60,6 +60,12 @@ gem "jwt"
 # Send app telemetry to Azure Application Insights
 gem "application_insights", git: "https://github.com/microsoft/ApplicationInsights-Ruby.git", ref: "5db6b4ad65262d23f26b678143a4a1fd7939e5c2"
 
+# Semantic Logger provides more useful application log entries
+gem "rails_semantic_logger"
+
+# Use logstash-logger to send to Logit.io
+gem "logstash-logger", "~> 0.26.1"
```

Comment:
I think this will restrict us to patch versions of `0.26`. Is there a particular reason why we should restrict the version? If the gem follows semver we could probably get away with using `~> 0.26` which allows us to use any version `< 1.0`